packages/kit/src/exports/public.d.ts

```diff
@@ -1045,7 +1045,7 @@ export interface Navigation {
 	to: NavigationTarget | null;
 	/**
 	 * The type of navigation:
-	 * - `form`: The user submitted a `<form>`
+	 * - `form`: The user submitted a `<form>` with a GET method
```

Comment:
I think this is a little confusing as it raises the question of what about a POST request 
```suggestion
	 * - `form`: The user submitted a `<form>` with a GET request
```

Reply:
it means 'with a `method` attribute whose value is `GET`'. agree it's a little ambiguous though — changing it to `<form method="GET">`